src/runtime/vm/executable.cc

```diff
@@ -525,12 +521,10 @@ void Executable::LoadConstantSection(dmlc::Stream* strm) {
   }
 
   // Load the const to device mapping.
-  std::vector<size_t> const_device_type;
+  std::vector<int64_t> const_device_type;
```

Comment:
to keep consistent with other places, let's just use Index for int64_t for this change, we have created the alias for it already.

Reply:
fixed